modules/ROOT/pages/silverblue-guide.adoc

```diff
@@ -179,7 +179,64 @@ NOTE: The prompts as displayed, are normal for my system setup. They will very l
+edit files manually and run commands from the terminal for now.
+
+==== Manual Upgrade Method
+To manually start an OS upgrade, in a terminal run the `sudo rpm-ostree -r upgrade`
+command to download any available upgrade, stage the upgrade files to the new
+image and then will reboot the computer. Remove the `-r` argument in the command
+if you want to manually reboot after the upgrade is staged instead.
+
+==== Setup Automatic Upgrades
+There is no graphical configuration tool available yet to setup automatic
+upgrades, so you need to edit files manually and run commands from the terminal
+to get it setup.
+
+First check what the current configuration is of automatic upgrade by running
+the `rpm-ostree status` command in the terminal. The first line of the output will show if
+automatic upgrade is enabled or not. If it says `AutomaticUpdates: disabled` then
+automatic upgrade is not enabled. If it says `AutomaticUpdates: fetch` then an upgrade will be
+downloaded but not staged. If it says `AutomaticUpdates: stage` the upgrade files
+will be downloaded and copied to the new image that will be activated after
+a reboot. To enable automatic ostree upgrade, do the following:
+
+1. Edit the file `/etc/rpm-ostreed.conf` and in the `[daemon]` section, uncomment
+and change the line that says
+`#AutomaticUpdatePolicy=none` to `AutomaticUpdatePolicy=fetch` to download the
+upgrade files only, or change it to `AutomaticUpdatePolicy=stage` to download
+and copy the files to the new OS image. Then save the file.
+
+2. Run the command `rpm-ostree reload` to make rpm-ostree aware of the configuration
+changes.
+
+3. Start the systemd unit timer by running command `sudo systemctl enable rpm-ostreed-automatic.timer`
+and then `sudo systemctl start rpm-ostreed-automatic.timer`.
+
+4. The default timer configuration is set to be triggered one hour after boot and
+then repeat every one day thereafter. To change the timer frequency, edit the file
+`/etc/systemd/system/timers.target.wants/rpm-ostreed-automatic.timer`
+
+5. You can check that the `rpm-ostreed-automatic.timer` is enabled by running the
+command `systemctl list-timers`. If it is in the output list, then it is enabled.
+
+==== Disable Automatic Upgrade
+To disable automatic upgrade, disable the `rpm-ostreed-automatic.service` with
+the command `sudo systemctl disable rpm-ostreed-automatic.service`.
+
 === Rolling Back the OS
+If something went wrong or is not working correctly with a new  upgrade, then it
+may be necessary to rollback the upgrade until a fix is available. To rollback
+to the previous OS deployment, run the command `sudo rpm-ostree -r rollback`.
+
+You can also deploy previous OS images that you have pinned. Use the
+`rpm-ostree status` command to see the list of previous pinned deployments that
+are available. Then use the `sudo rpm-ostree deploy [version|branch|commit-id]`
+command to deploy the pinned image from the list.
```

Comment:
This section is not entirely accurate.

You are able to deploy previous verseions that exist on the remote repo, not just those that have been pinned.

You can get a list of available versions by doing `ostree pull --commit-metadata-only --depth=-1 fedora:fedora/29/x86_64/silverblue` followed by `ostree log fedora:fedora/29/x86_64/silverblue`.  You can deploy any version or commit ID found in that log.

Pinning a deployment is a different operation; it is supposed to be used when you want to keep a particular version around after the host has been upgraded.  You can pin a deployment with `ostree admin pin`.

Please update this section with these details.  Thanks!